apps/gallery/.storybook/preview.tsx

```diff
@@ -0,0 +1,32 @@
+import { Controls, Description, Primary, Source, Stories, Subtitle, Title } from '@storybook/blocks'
+import React from 'react'
```

Comment:
Is react import needed? (might be global like in nextjs)

Reply:
Yes it is needed, otherwise you get an `ReferenceError: React is not defined` error